portal-impl/src/com/liferay/portal/service/impl/CompanyLocalServiceImpl.java

```diff
@@ -333,9 +333,22 @@ public Company checkCompany(String webId, String mx)
 			if (defaultUser != null) {
 				if (!defaultUser.isAgreedToTermsOfUse()) {
 					defaultUser.setAgreedToTermsOfUse(true);
+				}
+
+				defaultUser.setLanguageId(companyDefaultLocale.toString());
+
+				if (Validator.isNotNull(
```

Comment:
Why is this code necessary in this case?

I understand it might have been added for consistency with the code below, but what is the impact of adding it?